docs/subnets/create-a-fuji-subnet.md

```diff
@@ -803,6 +685,125 @@ After you update your config, you are going to need to restart your node for the
 take effect.
 ```
 
+## Add a Validator
+
+:::warning
+
+If the `join` command isn't successfully completed before `addValidator` is completed and the 
```

Comment:
Remove the `>20%` part. Just say the subnet may have degraded performance or even halt. Be scary haha.